source/Engine.cpp

```diff
@@ -951,6 +951,15 @@ void Engine::Draw() const
 			RingShader::Draw(pos, radius, 1.5f, it.disabled, color[6 + it.type], dashes, it.angle);
 	}
 
+	for(const shared_ptr<Ship> &ship : ships)
+	{
+		if(ship->Cloaking() && ship->IsYours() && ship->GetSystem() == player.GetSystem())
+		{
+			OutlineShader::Draw(ship->GetSprite(), (ship->Position()-center-ship->Velocity())*zoom, Point(ship->Width(), ship->Height())*zoom,
+								Color(0.6f, 0.1f, 0.1f, 0.8f*static_cast<float>(ship->Cloaking())), ship->Facing().Unit(), ship->GetFrame());
+		}
+	}
```

Comment:
You dont need any of these braces since its one line code within another one line code